src/audio/audio_stream.c

```diff
@@ -8,15 +8,15 @@
 
 static size_t audio_stream_get_free_size(struct sof_sink *sink)
 {
-	struct audio_stream *audio_stream = container_of(sink, struct audio_stream, sink_api);
+	struct audio_stream *audio_stream = container_of(sink, struct audio_stream, _sink_api);
 
 	return audio_stream_get_free_bytes(audio_stream);
 }
 
 static int audio_stream_get_buffer(struct sof_sink *sink, size_t req_size,
 				   void **data_ptr, void **buffer_start, size_t *buffer_size)
 {
-	struct audio_stream *audio_stream = container_of(sink, struct audio_stream, sink_api);
+	struct audio_stream *audio_stream = container_of(sink, struct audio_stream, _sink_api);
```

Comment:
I suppose you could take this 1 step further in the future by having a static inline accessor API that wraps ```container_of()```